site/_posts/2017-04-03-Announcing-Fine-Beta2-Release.html.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Announcing Fine Beta Release
+author: chessbyte
+date: 2017-05-01 07:49:24 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, we're pleased to announce the Beta 2 release for ManageIQ Fine! You can download the Fine Beta 2 Release [here](http://manageiq.org/download/). You'll find it under the "Pre-release (Fine-1-Beta2)" section.
```

Comment:
> You'll find it under the "Pre-release (Fine-1-Beta2)" section.

A bit redundant, and also the sections on the download page change with each release. I'd leave this line out but it's up to you.